Patch-Source: https://salsa.debian.org/perl-team/interpreter/perl/-/raw/debian/5.40.0-3/debian/patches/fixes/json-pp-options.diff
--
From: Niko Tyni <ntyni@debian.org>
Date: Wed, 7 Aug 2024 22:53:54 +0300
Subject: Fix json_pp option handling

Getopt-Long ignores case by default, so -v never worked.

Getopt-Long 2.55 / Perl 5.40 started to warn about this.

Bug: https://github.com/makamaka/JSON-PP/issues/88
Bug-Debian: https://bugs.debian.org/1078058
---
 cpan/JSON-PP/bin/json_pp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/cpan/JSON-PP/bin/json_pp b/cpan/JSON-PP/bin/json_pp
index 72b9db7..85dd348 100644
--- a/cpan/JSON-PP/bin/json_pp
+++ b/cpan/JSON-PP/bin/json_pp
@@ -2,7 +2,7 @@
 
 BEGIN { pop @INC if $INC[-1] eq '.' }
 use strict;
-use Getopt::Long;
+use Getopt::Long qw( :config no_ignore_case );
 use Encode ();
 
 use JSON::PP ();
